core/src/main/java/org/bitcoinj/evolution/SimplifiedMasternodeListManager.java

```diff
@@ -121,9 +121,6 @@ public enum SaveOptions {
     long tipHeight;
     Sha256Hash tipBlockHash;
 
-    //AbstractBlockChain blockChain;
-    //AbstractBlockChain headersChain;
-
```

Comment:
Some of these changes were pointed out in a previous coinjoin PR.  Those are being fixed here and will be later merged with the `feature-coinjoin` branch.